Gemfile

```diff
@@ -31,7 +31,7 @@ gem 'redis', '~> 4.0'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 gem 'devise', '>= 4.7.1'
-gem 'devise_saml_authenticatable', git: 'https://github.com/apokalipto/devise_saml_authenticatable', branch: 'master'
+gem 'devise_saml_authenticatable', git: 'https://github.com/apokalipto/devise_saml_authenticatable', ref: 'fc398ff'
```

Comment:
I don't remember...why the switch to a specific commit?

Reply:
Jalpa suggested it so it doesn't brake on bundle install. Therefore, you know exactly where it is at across all servers.